hooks/src/lib/util/validateCardNumber/validateCardNumber.ts

```
@@ -0,0 +1,55 @@
+  cardNumber: string,
+  cardNetwork: CardNetwork,
+  index: number
+) {
+  if (cardNumber.length === 0) return "";
+
+  if (!isNumeric(cardNumber)) {
+    return ERROR_MESSAGE.NOT_NUMERIC;
+  }
+
+  if (
+    !CARD_INPUT_LENGTH.hasOwnProperty(cardNetwork) ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)